build/build.sh

```diff
@@ -69,12 +69,12 @@ moveCompiledBuildx(){
 buildx(){
     output_name="bin/$CTLNAME"
     echo "Building for: ${GOOS} ${GOARCH}"
-    gox -cgo -os="$GOOS" -arch="$GOARCH" -ldflags \
-        "-X github.com/openebs/node-disk-manager/pkg/version.GitCommit=${GIT_COMMIT} \
-        -X main.CtlName='${CTLNAME}' \
-        -X github.com/openebs/node-disk-manager/pkg/version.Version=${VERSION}" \
-        -output="$output_name" \
-        ./cmd/"$BUILDPATH"
+     go build \
+     -ldflags="-X github.com/openebs/node-disk-manager/pkg/version.GitCommit=${GIT_COMMIT} \
```

Comment:
nit: Indentation needs to be maintained for multiline commands. A single tab or 2 spaces will be fine,